diff -Nur -x '*.orig' -x '*~' pycam-0.2.2//setup.py pycam-0.2.2.new//setup.py
--- pycam-0.2.2//setup.py	2010-03-20 12:57:16.000000000 +0100
+++ pycam-0.2.2.new//setup.py	2010-03-20 13:07:53.026518970 +0100
@@ -43,7 +43,7 @@
         "pycam.Toolpath",
         "pycam.Physics",
     ],
-    scripts = ['pycam', 'pycam_win32_postinstall.py'],
+    scripts = ['pycam'],
     data_files=[("share/doc/pycam",[
             "COPYING.TXT",
