diff --git a/setup.py b/setup.py
index 3046304..9f66f6d 100755
--- a/setup.py
+++ b/setup.py
@@ -45,13 +45,9 @@ setup(
     ],
     scripts = ['pycam', 'pycam_win32_postinstall.py'],
     data_files=[("share/doc/python-pycam",[
-            "COPYING.TXT",
             "HOWTO.TXT",
-            "INSTALL.TXT",
-            "LICENSE.TXT",
             "README.TXT",
-            "Changelog",
-            "release_info.txt"]),
+            "Changelog"]),
         ("share/python-pycam/ui", [
             os.path.join("share", "gtk-interface", "pycam-project.ui"),
             os.path.join("share", "gtk-interface", "menubar.xml"),
